### ISSUE TITLE
[FEAT]: Add command to change server URL, delete drives and clear history

### ISSUE THREAD

**Issue by gamemaker1** [gamemaker1/dabbu-cli#64]
**Is your feature request related to a problem? Please describe.**
When running the server on my machine, the server keeps using a different port number every time. It would be easy for me to change the server URL using a command instead of editing the config file manually.

**Describe the solution you'd like**
Add a `config` command that allows you to edit the server URL, delete drives and delete command history. Example usage:

```
$ config set serverUrl "http://localhost:23983"
$ config del history
$ config del drives.c
```

**Comment by gamemaker1** [comment]
Note that I can make the server run on a fixed port by setting the PORT environment variable through terminal (`env PORT=8000 /path/to/file`), but some may prefer running it by double clicking it in file manager.

**Comment by gamemaker1** [comment]
Closed by commit 4e2baff (adds config command)
